.github/workflows/build-dev-image.yml

```
@@ -0,0 +1,127 @@
+          echo "## 🐳 Dev Docker Image Built Successfully" >> $GITHUB_STEP_SUMMARY
+          echo "" >> $GITHUB_STEP_SUMMARY
+          echo "**Branch:** ${{ inputs.branch }}" >> $GITHUB_STEP_SUMMARY
+          echo "**Commit:** ${{ github.sha }}" >> $GITHUB_STEP_SUMMARY
+          echo "**Image:** \`guarzo/wanderer-notifier:dev\`" >> $GITHUB_STEP_SUMMARY
+          echo "" >> $GITHUB_STEP_SUMMARY
+          echo "### Usage" >> $GITHUB_STEP_SUMMARY
+          echo "\`\`\`bash" >> $GITHUB_STEP_SUMMARY
+          echo "docker pull guarzo/wanderer-notifier:dev" >> $GITHUB_STEP_SUMMARY
+          echo "\`\`\`" >> $GITHUB_STEP_SUMMARY
+          echo "" >> $GITHUB_STEP_SUMMARY
+          echo "⚠️ **Note:** This dev image will be overwritten by subsequent dev builds from any branch." >> $GITHUB_STEP_SUMMARY
```

YAMLlint (1.37.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)